src/pages/comparison/components/rate.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import './rate.css';
+
+export const Rate = (props: { rate: number}) => {
+
+    let clases: string[] = [];
+
+    switch (props.rate) {
+        case 1: {
+            clases = ['r', 'gy', 'gy','gy', 'gy']; 
+            break;
+        }
+        case 2: {
+            clases = ['y', 'y', 'gy', 'gy', 'gy'];
+            break;
+        }
+        case 3: {
+            clases = ['y', 'y', 'y', 'gy', 'gy'];
+            break;
+        }
+        case 4: {
+            clases = ['gr', 'gr', 'gr', 'gr', 'gy'];
+            break;
+        } 
+        case 5: {
+            clases = ['gr', 'gr', 'gr', 'gr', 'gr'];
+            break;
+        }
+        default: break;
+    }
+
+    return (
+        <span className="rate-bar">
+            <div className={clases[0]}></div>
+            <div className={clases[1]}></div>
+            <div className={clases[2]}></div>
+            <div className={clases[3]}></div>
+            <div className={clases[4]}></div>
+        </span>
```

Comment:
Seems that divs inside span is not the best choice.
Would you like to consider using loop in this case?

Reply:
After so much work on the React, you no longer pay attention to html :)

Actually i had no idea how to use loop in this case, but i will think about it. Maybe you solve this issue much better :)